decompiler/config/all-types.gc

```diff
@@ -37561,10 +37551,10 @@
 ;; - Unknowns
 
 (define-extern *sun-iris-door-sg* skeleton-group)
-;;(define-extern sun-iris-door-open object) ;; unknown type
-;;(define-extern sun-iris-door-closed object) ;; unknown type
-;;(define-extern sun-iris-door-opening object) ;; unknown type
-;;(define-extern sun-iris-door-closing object) ;; unknown type
+(define-extern sun-iris-door-open state)
+(define-extern sun-iris-door-closed state)
+(define-extern sun-iris-door-opening state)
+(define-extern sun-iris-door-closing state)
```

Comment:
Can we make these `(state sun-iris-door)` or does that cause decompiler issues?

Reply:
This still works the same as before, I just wanted to check the top-level code for states on an actual random game object.